scripts/s3/s3api/bucket_operations_negative.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+"""S3 bucket operation workload for io stability."""
+import random
+import string
+from datetime import datetime
+
+from src.commons.exception import CreateBucketException, \
+    UploadObjectException, \
+    ListObjectException, \
+    DeleteBucketException, \
+    HeadBucketException
+
+from src.commons.constants import MIN_DURATION
+from scripts.s3.s3api.bucket_operations import TestBucketOps
+
+INVALID_BUCKET = "Invalid bucket url: {%s}\nException: {%s}"
+ERROR_CODE_RESPONSE = 'Error Code: %s Error Message: %s'
+
+
+def create_invalid_bucket_name():
+    """Create invalid bucket name which should not fulfill following criteria
+
+    Between 3 and 63 characters long, and can contain lower-case characters,
+    Can contain numbers,periods, and dashes.
+    Each label in the bucket name must start with a lowercase letter or number.
+    Cannot contain underscores, end with a dash,
+    Cannot have consecutive periods, or use dashes adjacent to periods.
+    The bucket name cannot be formatted as an IP address (198.51.100.24)."""
+    name_length = random.randint(1, 64)
+    return ''.join(random.choices(string.ascii_letters + string.digits, k=name_length))
```

Comment:
This will generate valid bucket name as lower case letters + digits are supported.

Reply:
Done updates